Rawl_B502/genelach_úa_Fáeláin.ttl

```diff
@@ -0,0 +1,336 @@
+
+<#Daimín>
+    a foaf:Person;
+    irishRel:genName "Daimín";
+    rel:childOf <#Muiredach>.
+
+<#MáelCarmain>
+    a foaf:Person;
+    irishRel:nomName "Máel Carmain";
+    rel:childOf <#Murchada-53f6ef70>;
+    rel:childOf <#Scirddech>
+
+<#Scirddech>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Scirddech";
+    rdfs:comment "banamus".
+
+<#Gormlaith>
+    a foaf:Person;
+    irishRel:nomName "Gormlaith";
+    foaf:gender "female";
+    rel:childOf <#Murchada-53f6ef70>;
+    rel:spouseOf <#Briain>;
+    owl:sameAs <http://example.com/LL/síl_daimini.ttl#Gormlaith>.
+
+<#Briain>
+    a foaf:Person;
+    irishRel:genName "Briain";
+    irishRel:nomName "Brian";
+    owl:sameAs <http://example.com/LL/de_rigaib_muman_iar_cretim_inso.ttl#Brian-da4b783d>.
+
+<#Fáelan>
+    a foaf:Person;
+    irishRel:nomName "Fáelan";
+    owl:sameAs <#Fáeláin>;
+    irishRel:numChild 2.
+
+<#Cerball>
+    a foaf:Person;
+    irishRel:nomName "Cerball";
+    owl:sameAs <#Cerbaill-394aeeb0>;
+    rel:childOf <#Fáelan>.
+
+<#GlúnIarn>
+    a foaf:Person;
+    irishRel:nomName "Glún Iarn";
+    rel:childOf <#Fáelan>.
+
+<#Cerbaill-8d71a820>
+  irishRel:numChild 3;
+  owl:sameAs <#Cerball>.
+
+<#Donnchad>
+    a foaf:Person;
+    irishRel:nomName "Donnchad";
+    rel:childOf <#Cerbaill-8d71a820>.
+
+<#Murchad>
+    a foaf:Person;
+    irishRel:nomName "Murchad";
+    rel:childOf <#Cerbaill-8d71a820>.
+#This seems like a scribal error for Máel Morda? EPT
+
+<#GillaPátraic>
+    a foaf:Person;
+    irishRel:nomName "Gilla Pátraic";
+    rel:childOf <#Cerbaill-8d71a820>.
+
+<#Find-8b9aa8c0>
+    a foaf:Person;
+    irishRel:nomName "Find";
+    irishRel:numChild 2;
+    owl:sameAs <#Find>.
+
+<#DomnallFeccach>
+    a foaf:Person;
+    irishRel:nomName "Domnall Feccach";
+    rel:childOf <#Find-8b9aa8c0>.
+
+<#Murchad-e97944b0>
+    a foaf:Person;
+    irishRel:nomName "Murchad";
+    rel:childOf <#Find-8b9aa8c0>;
+    owl:sameAs <#Murchada-53f6ef70>.
+
+<#GlúnIairn>
+    a foaf:Person;
+    irishRel:nomName "Glún Iairn";
+    rel:childOf <#Cellaich>.
+
+<#Cerball-78830150>
+    a foaf:Person;
+    irishRel:nomName "Cerball";
+    rel:childOf <#Cellaich>.
+
+<#Cellaich>
+    a foaf:Person;
+    irishRel:nomName "Cellach";
+    irishRel:genName "Cellaich";
+    rel:childOf <#Bróen>.
+
+<#Bróen>
+    a foaf:Person;
+    irishRel:nomName "Bróen";
+    irishRel:genName "Bróen";
+    rel:childOf <#MáelMórda>.
+
+<#MáelMórda-fa0c83d0>
+    a foaf:Person;
+    irishRel:nomName ;
+    irishRel:genName "Máel Mórda";
+    rel:childOf <#Murchada-ebc93d40>.
+
+<#Murchada-ebc93d40>
+    a foaf:Person;
+    irishRel:nomName "Murchad";
+    irishRel:genName "Murchada";
+    rel:childOf <#Finn>;
+    owl:sameAs <#Murchada-53f6ef70>.
+
+<#Finn>
+    a foaf:Person;
+    irishRel:nomName "Finn";
+    irishRel:genName "Finn";
+    rel:childOf <#MáelMórdai>;
+    owl:sameAs <#Find>.
+
+<#MáelMórdai>
+    a foaf:Person;
+    irishRel:nomName ;
```

Comment:
I think this should be irishRel:nomName "Máel Mórda" but please make sure I am correct :)